torchaudio/models/decoder/_ctc_decoder.py

```diff
@@ -293,44 +293,6 @@ def ctc_decoder(
     )
 
 
-def lexicon_decoder(
```

Comment:
Could you remind me what was the reason behind of the removal? Is there an alternative API?

Reply:
I think this is included in `ctc_decoder`, by adding `lexicon` to it.

Reply:
`lexicon_decoder` functionality is now part of `ctc_decoder`. The function was left when we changed the naming/function to not break anyone's code who was previously using `lexicon_decoder`. In the beta stage we only need to keep `ctc_decoder` as `torchaudio.models.decoder.lexicon_decoder` is not something that people ever used. I added back the deprecation warning in prototype for both of these functions [here](https://github.com/pytorch/audio/pull/2410/commits/b66abbc04fae44d4ef545d0499aece127cc1651e) so that previous code using these functions don't break

Reply:
sounds good, thanks for the clarification.